Sources/Sentry/include/SentryLog.h

```diff
@@ -43,7 +43,7 @@ NS_ASSUME_NONNULL_END
 #define SENTRY_LOG_ERRNO(statement)                                                                \
     ({                                                                                             \
         errno = 0;                                                                                 \
-        const auto __log_rv = (statement);                                                         \
```

Comment:
This was the other thing that wasn't working correctly. This macro used to only be called from C++ contexts, but now that it's called from the Objective-C logger to initialize the async clogger, we can't use the auto keyword.